.clang-tidy

```diff
@@ -39,9 +50,10 @@ modernize-*,
 performance-*,
 -performance-noexcept-move-constructor,
 -performance-unnecessary-value-param,
+-performance-no-int-to-ptr,
 '
 HeaderFilterRegex: '^(c10/(?!test)|torch/csrc/(?!deploy/interpreter/cpython)).*$'
 AnalyzeTemporaryDtors: false
 WarningsAsErrors: '*'
-CheckOptions:
+CheckOptions: []
```

Comment:
I don't think this is right.... won't this cause all the other check options to be ignored (that lintrunner use to replace the ... with?)

Reply:
Can we delete this line?